README.md

```diff
@@ -1,12 +1,11 @@
 #Business
 C4GNV is a member brigade of Code for America, the goal of this project is to make it easier
 to start a business in Gainesville by providing people easy access to permits and zoning information.
-The project is a light weight website written in Typescript and Angular 2, using open data sources
-as a persistent backend.
+The project is a lightweight website written in [Typescript](https://www.typescriptlang.org/) and Angular 2, using open data sources as a persistent backend.
 
-The inspiration for this service is to eventually be able to expand to be like the
```

Comment:
Why remove "the" here? Is it not correct to say "to be like the Business Portal for San Francisco" and instead turn it into "to be like Business Portal for San Franscisco"...the second one just sounds wrong.